packages/lambda-functions/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@coveo/lambda-functions",
```

Comment:
`@coveo/search-token-serverless` ?

Reply:
I like @olamothe proposal, here's my take on it too 😄 
`@coveo/search-token-lambda`?